src/components/BSelect/BSelect.vue

```diff
@@ -23,12 +23,12 @@
       <span class="dropdown-arrow"><i class='icon ion-arrow-down-b' aria-hidden='true'></i></span>
     </button>
     <ul
+      v-show="opened"
```

Comment:
Loading them to the page and hiding/showing instead of adding and removing them from the DOM fixed that problem?